packages/core/test/session.test.ts

```diff
@@ -1,6 +1,4 @@
-const { describe, it, expect } = global
-
-const Session = require('../session')
+import Session from '../session'
```

Comment:
This is an example where the JS infer has worked in our favour. `toJSON()` and `_track` are not defined in the manual type definitions but have been well-inferred from the JS and so we have no type issues.